### PR TITLE
[CHI-204] Add Fieldset template

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

**Comment by davidojedacl** [comment]
We make this with grid columns -noGutter and include -mr--1 to match with the design prototype because the default gutter of the grid system are higher. In the image the grid columns system with the defaut gutter padding 24px
Screen Shot 2019-03-14 at 2 50 14 PM

File: src/website/views/templates/fieldset.pug [hunk 1]
```diff
@@ -0,0 +1,218 @@
+---
+title: Fieldset
+---
+
+p.-text Basic example of several types of form elements in one fieldset.
+
+.example.-mb--3
+  .chi-app.-py--3.-bg--grey-20(style="border-radius:.15em .15em 0 0;")
+    .a-grid__container
+      .a-grid.-noGutter.-b--1.-p--3.-bg--white
```
Comment (mattnickles) [inline src/website/views/templates/fieldset.pug]: Can you remove `-b--1` and `-bg--white` and replace with `a-card`?

Comment (jllr) [reply src/website/views/templates/fieldset.pug]: can you add -flex--row class too?

File: src/website/views/templates/fieldset.pug [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+            label(for='email') Email
+        div.a-col.-w--12
+          h6.-mt--2 Executive contact
+        div.a-col.-w--12
+          .m-formGroup.-mb--2
+            input(type="checkbox", class="a-input", id="checkbox2" checked)
+            label(for="checkbox2")
+            label.-text(for="checkbox2") Same as business contact
+        div.a-col.-w--12
+          h6.-mt--2 Additional business information
```
Comment (mattnickles) [inline src/website/views/templates/fieldset.pug]: Remove `-mt--2` - it's setting a margin that's inconsistent with the space between the checkbox and h6 above it.

Comment (davidojedacl) [reply src/website/views/templates/fieldset.pug]: done

File: src/website/views/templates/fieldset.pug (hunk 1 above)
Comment (davidojedacl) [reply src/website/views/templates/fieldset.pug]: done

**Pull request merged**